remme/rpc_api/event/_handlers.py

```diff
@@ -164,12 +165,21 @@ def parse_evt(self, evt):
 
     def validate(self, msg_id, params):
         try:
-            id = params['id']
+            batch_id = params['id']
+
         except KeyError:
             raise RpcInvalidParamsError(message='Missed id', msg_id=msg_id)
 
+        if len(batch_id) != 128:
```

Comment:
this to check could be united, client do not need to know, what regex and what he does not match, just generalized - `Incorrect id`

Reply:
@DeV1doR, done.